I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 182, in close_tts
    audio_generator.exit()
AttributeError: module 'audio_generator' has no attribute 'exit'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "C:\Users\water\Desktop\renpy-8.2.1-sdk\renpy\main.py", line 690, in main
    i()
  File "game/script.rpy", line 182, in close_tts
    audio_generator.exit()
AttributeError: module 'audio_generator' has no attribute 'exit'

Windows-10-10.0.19041 AMD64
Ren'Py 8.3.2.24090902
Ushio_Noa 0.1.0_beta
Mon Jan  6 20:17:12 2025
